### PR TITLE
feat: add lido-logo component

### PR DESCRIPTION
**Pull request by solidovic**

### CODE REVIEW

File: packages/lido-logo/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Lido Logo Component
+
+React component for Lido Finance projects.
+Part of [Lido UI Components](https://github.com/lidofinance/ui/#readme)
+
+## Install
+
+```bash
+yarn add @lidofinance/block
```
Comment (avsetsin) [inline packages/lido-logo/README.md:9]:
```suggestion
yarn add @lidofinance/lido-logo
```

File: packages/lido-logo/src/lido-logo.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...0 0 0-1.331-.3h-1.5v7.5h1.5zM82.558 16a5.56 5.56 0 0 1-2.213-.45 5.978 5.978 0 0 1-1.818-1.275 6.625 6.625 0 0 1-1.238-1.912A6.118 6.118 0 0 1 76.84 10c0-.838.15-1.619.45-2.344a6.304 6.304 0 0 1 1.238-1.912 5.575 5.575 0 0 1 1.818-1.275A5.367 5.367 0 0 1 82.558 4a5.24 5.24 0 0 1 2.194.469c.7.3 1.306.725 1.818 1.275a5.868 5.868 0 0 1 1.238 1.912c.312.725.469 1.506.469 2.344 0 .85-.157 1.637-.47 2.363a6.144 6.144 0 0 1-1.237 1.912 5.689 5.689 0 0 1-1.818 1.275c-.688.3-1.42.45-2.194.45zm0-2.156c.462 0 .906-.094 1.331-.281a3.31 3.31 0 0 0 1.106-.788 4.13 4.13 0 0 0 .75-1.219c.188-.475.282-.993.282-1.556 0-.563-.094-1.075-.282-1.537a3.855 3.855 0 0 0-.75-1.22 3.263 3.263 0 0 0-2.437-1.088 3.333 3.333 0 0 0-1.35.282 3.25 3.25 0 0 0-1.088.807 3.854 3.854 0 0 0-.75 1.219A4.057 4.057 0 0 0 79.09 10c0 .563.093 1.081.28 1.556.188.463.438.869.75 1.219.313.338.675.6 1.088.787a3.3 3.3 0 0 0 1.35.282z"/><path fill-rule="evenodd" clip-rule="evenodd" d="m7.005 0 4.949 7.748-4.949 2.884-4.948-2.884L7.005 0zM3.572 7.381l3.433-5.376 3.434 5.376-3.434 2.002L3.572 7.38z" fill="#00A3FF"/><path d="M6.998 12.335 1.258 8.99l-.157.245a6.999 6.999 0 1 0 11.796 0l-.157-.245-5.742 3.346z" fill="#00A3FF"/></svg>
```
Comment (avsetsin) [inline packages/lido-logo/src/lido-logo.svg:1]: This file is not used anywhere

File: .idea/codeStyles/Project.xml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (avsetsin) [inline .idea/codeStyles/Project.xml:1]: Is the .idea directory needed in the repository? Are these files not automatically generated?

Comment (solidovic) [reply .idea/codeStyles/Project.xml:1]: not need. i will delete it

File: packages/lido-logo/src/lido-logo.svg (hunk 2 above)
Comment (solidovic) [reply packages/lido-logo/src/lido-logo.svg:1]: i will delete it

File: packages/lido-logo/README.md (hunk 1 above)
Comment (solidovic) [reply packages/lido-logo/README.md:9]: ok

File: .idea/codeStyles/Project.xml (hunk 3 above)
Comment (solidovic) [reply .idea/codeStyles/Project.xml:1]: yes, this files auto created

**Review by avsetsin — APPROVED** [review]
lgtm

**Pull request merged**